src/sentry/tasks/auto_ongoing_issues.py

```diff
@@ -34,7 +34,9 @@ def schedule_auto_transition() -> None:
                 id=Project.objects.get_from_cache(id=project_id).organization_id
             )
         except Exception:
-            org = Organization.objects.get(project_id=project_id)
+            org = Organization.objects.get(
```

Comment:
I'm not sure that there's any reason to do this `except`, we'll fall back to fetching from the databse here anyway https://github.com/getsentry/sentry/blob/0db44d6460577e5a9ae25ed59bae630f218089bf/src/sentry/db/models/manager/base.py#L293-L301

Unless there's some specific case you noticed here?

Reply:
Nah, no specific case - was mostly being overly cautious, but if we're raising exceptions here its probably better to fix it than reattempt to lookup the org in a non-optimal way anyways.